libs/libvtrcapnproto/matrix.capnp

```diff
@@ -0,0 +1,9 @@
+@0xafffc9c1f309bc00;
+
```

Comment:
I assume we can add comments to the schema files?

It would be good to give a comment which tells the reader this corresponds to serializing a vtr::NdMatrix. It would also be good to comment what the various fields represent so someone less familiar with capnp could get the jist of it.

Reply:
Sure